docs/how_to/use_estimator.rst

```diff
@@ -46,6 +46,10 @@ Then you need to create the :class:`~qiskit.circuit.QuantumCircuit` for which yo
     qc.h(0)
     qc.cx(0,1)
 
+.. note::
+
+    The :class:`~qiskit.circuit.QuantumCircuit` can't include any measurements.
```

Comment:
Taken at face value this is an incorrect statement. A QuantumCircuit can include measurements. If however you are going to pass it to an Estimator, the case here, then it must not include any measurement instructions.

Reply:
You're right. How about this?

> The ``QuantumCircuit`` you pass to ``Estimator`` must not include any measurement
> The ``QuantumCircuit`` you pass to ``Sampler`` must include measurements

Reply:
Updated in c889e90.